### ISSUE TITLE
Blu-ray.com estimator crashes when info not found

### ISSUE THREAD

**Issue by Tommatheussen** [Flexget/Flexget#1353]
### Expected behaviour:
Flexget shouldn't crash.

### Actual behaviour:
Flexget crashes when certain info can't be found

### Steps to reproduce:
 - Step 1: Make sure a movie is in a movie_list (or any input) that does not exist on blu-ray.com. (for example The Seamstress(2009)
 - Step 2: Use a task that tries get the release dat for the movie.

#### Config:
```
DownloadMoviesTest:
    discover:
      interval: 3 hour
      what:
        - movie_list:
            list_name: "Testing"
      from:
        - piratebay: yes
```
  
#### Log:
```
2016-08-18 11:51 DEBUG    est_released  DownloadMoviesTest The Seamstress (2009)
2016-08-18 11:51 DEBUG    est_movies_bluray DownloadMoviesTest Searching Blu-ray.com for release date of The Seamstress (2009)
2016-08-18 11:51 VERBOSE  api_bluray    DownloadMoviesTest Searching from blu-ray.com title=The Seamstress
2016-08-18 11:51 DEBUG    utils.requests DownloadMoviesTest Fetching http://m.blu-ray.com/quicksearch/search.php
2016-08-18 11:51 DEBUG    utils.requests DownloadMoviesTest Fetching http://m.blu-ray.com/countries.json.php
2016-08-18 11:51 DEBUG    utils.requests DownloadMoviesTest Fetching http://m.blu-ray.com/movies/The-Seamstress-Blu-ray/6012/
2016-08-18 11:51 CRITICAL task          DownloadMoviesTest BUG: Unhandled error in plugin discover: 'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 472, in __run_plugin
    return method(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/input/discover.py", line 289, in on_task_input
    entries = self.estimated(entries, estimation_mode)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/input/discover.py", line 201, in estimated
    est_date = estimator.estimate(entry)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/estimators/est_released.py", line 28, in estimate
    estimate = estimator(entry)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/estimators/est_released_movies_bluray.py", line 31, in estimate
    movie = lookup(title=movie_name, year=movie_year, session=session)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/internal/api_bluray.py", line 209, in lookup
    movie = BlurayMovie(title=title, year=year)
  File "<string>", line 4, in __init__
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/state.py", line 306, in _initialize_instance
    manager.dispatch.init_failure(self, args, kwargs)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/util/langhelpers.py", line 60, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/state.py", line 303, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/internal/api_bluray.py", line 121, in __init__
    self.rating = float(movie_info.find('div', id='ratingscore').text.strip())
AttributeError: 'NoneType' object has no attribute 'text'
```

### Additional information:

- Flexget Version: 2.3.0
- Python Version: 2.7.12
- Installation method: pip
- OS and version: Ubuntu server 14.04
- Link to crash log: http://pastebin.com/1bEAkpj6

**Comment by cvium** [comment]
fixed in fb6c2c3

**Comment by lostsubstance** [comment]
I just encountered this crash

```
2016-08-24 23:24 DEBUG    est_released  get-movies      A Troll in Central Park (1994)                                                                                                           2016-08-24 23:24 DEBUG    est_movies_bluray get-movies      Searching Blu-ray.com for release date of A Troll in Central Park (1994)                                                             2016-08-24 23:24 VERBOSE  api_bluray    get-movies      Searching from blu-ray.com title=A Troll in Central Park                                                                                 2016-08-24 23:24 DEBUG    utils.requests get-movies      Fetching http://m.blu-ray.com/quicksearch/search.php                                                                                    2016-08-24 23:24 DEBUG    utils.requests get-movies      Fetching http://m.blu-ray.com/countries.json.php                                                                                        2016-08-24 23:24 DEBUG    utils.requests get-movies      Fetching http://m.blu-ray.com/movies/A-Troll-in-Central-Park-and-Thumbelina-Blu-ray/58752/                                              2016-08-24 23:24 CRITICAL task          get-movies      BUG: Unhandled error in plugin discover: 'NoneType' object has no attribute 'text'                                                       2016-08-24 23:24 CRITICAL manager       get-movies      An unexpected crash has occurred. Writing crash report to C:\Users\Jonathan\flexget\crash_report.2016.08                                 .24.232403370000.log. Please verify you are running the latest version of flexget by using "flexget -V" from CLI or by using version_checker plugin at http://fl                                 exget.com/wiki/Plugins/version_checker. You are currently using version 2.3.5                                                                                                                    2016-08-24 23:24 DEBUG    manager       get-movies      Traceback:                                                                                                                               Traceback (most recent call last):                                                                                                                                                                 File "c:\python27\lib\site-packages\flexget\task.py", line 472, in __run_plugin                                                                                                                    return method(*args, **kwargs)                                                                                                                                                                 File "c:\python27\lib\site-packages\flexget\event.py", line 23, in __call__                                                                                                                        return self.func(*args, **kwargs)                                                                                                                                                              File "c:\python27\lib\site-packages\flexget\plugins\input\discover.py", line 289, in on_task_input                                                                                                 entries = self.estimated(entries, estimation_mode)                                                                                                                                             File "c:\python27\lib\site-packages\flexget\plugins\input\discover.py", line 201, in estimated                                                                                                     est_date = estimator.estimate(entry)                                                                                                                                                           File "c:\python27\lib\site-packages\flexget\plugins\estimators\est_released.py", line 28, in estimate                                                                                              estimate = estimator(entry)                                                                                                                                                                    File "c:\python27\lib\site-packages\flexget\plugins\estimators\est_released_movies_bluray.py", line 31, in estimate                                                                                movie = lookup(title=movie_name, year=movie_year, session=session)                                                                                                                             File "c:\python27\lib\site-packages\flexget\plugins\internal\api_bluray.py", line 217, in lookup                                                                                                   movie = BlurayMovie(title=title, year=year)                                                                                                                                                    File "<string>", line 4, in __init__                                                                                                                                                             File "c:\python27\lib\site-packages\sqlalchemy\orm\state.py", line 306, in _initialize_instance                                                                                                    manager.dispatch.init_failure(self, args, kwargs)                                                                                                                                              File "c:\python27\lib\site-packages\sqlalchemy\util\langhelpers.py", line 60, in __exit__                                                                                                          compat.reraise(exc_type, exc_value, exc_tb)                                                                                                                                                    File "c:\python27\lib\site-packages\sqlalchemy\orm\state.py", line 303, in _initialize_instance                                                                                                    return manager.original_init(*mixed[1:], **kwargs)                                                                                                                                             File "c:\python27\lib\site-packages\flexget\plugins\internal\api_bluray.py", line 135, in __init__                                                                                                 self.overview = contents.find('p').text.strip()                                                                                                                                              AttributeError: 'NoneType' object has no attribute 'text'                                                                                                                                        2016-08-24 23:24 WARNING  task          get-movies      Aborting task (plugin: discover) 
```
